authgear/templates/usage-record-cronjob.yaml

```diff
@@ -12,7 +12,7 @@ data:
     authgear-portal usage collect-count active-user this-month
     authgear-portal usage collect-count sms-sent yesterday
     authgear-portal usage collect-count email-sent yesterday
-    authgear-portal usage collect-count whatsapp-sent yesterday
+    authgear-portal usage collect-count {{ include "authgear.usageRecord.whatsapp.event" }} yesterday
```

Comment:
Shouldn't it be `{{ .Values.authgear.usageRecord.whatsapp.event }}`?

Reply:
Ah yes sorry made a mistake